romana-install/roles/provision/create-aws-hosts/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Generate CloudFormation Template
+  template: src="aws_cf_small_cluster.json" dest="{{ stack_data_dir }}/cloudformation.json"
```

Comment:
Give it a try and see how it works. Would be nice to keep the download directory clean.

I usually create those temp files in /tmp even.